packages/performance/src/resources/trace.ts

```diff
@@ -262,6 +250,22 @@ export class Trace implements PerformanceTrace {
     }
   }
 
+  /**
+   * Converts the provided value to an integer value to be used in case of a metric.
+   * @param providedValue Provided number value of the metric that needs to be converted to an integer.
+   *
+   * @returns Converted integer number to be set for the metric.
+   */
+  private convertMetricValueToInteger(providedValue: number): number {
```

Comment:
This does not have to be a member of the class and can be added to metrics_utils file.

Reply:
Makes sense Ali. Thanks for the suggestion, done.